bindings/python/Makefile.am

```diff
@@ -14,17 +14,14 @@ dist-hook:
 	cp -R $(srcdir)/* $(distdir)/tmp
 	rm -f $(distdir)/tmp/ioprocess/config.py  # Must be regenerated
 	chmod -R a+rw $(distdir)/tmp
-	$(PYTHON) $(distdir)/tmp/setup.py --verbose sdist --dist-dir "`readlink -f $(distdir)`" --manifest "`readlink -f $(distdir)`/MANIFEST"
+	$(PYTHON) -m build --sdist --verbose --no-isolation --outdir "`readlink -f $(distdir)`"
 	rm -rf $(distdir)/tmp
 	tar -xvf $(distdir)/ioprocess-$(VERSION).tar* -C "$(distdir)" --strip-components=1
 	rm $(distdir)/ioprocess-$(VERSION).tar*
 
 install-data-local:
 	$(PYTHON) "$(srcdir)/setup.py" build -b "`readlink -f $(builddir)`" install --root "$(DESTDIR)/" --prefix "$(prefix)" clean -a
 
-uninstall-local:
-	rm -r `$(PYTHON) -c 'from distutils import sysconfig; print(sysconfig.get_python_lib(0,0,"$(DESTDIR)${prefix}"))'`/ioprocess*
```

Comment:
Can't we replace it with a non-disutils variant?

Reply:
It's possible but a bit complicated. Do we really need it?

Reply:
`'import sysconfig; print(sysconfig.get_path("purelib").replace("/usr", "/usr/local"))'` something like this should do.
Dont know if it's really used, but well :)